news/_posts/releases/2021-11-16-prestashop-1-7-8-1-maintenance-release.md

```diff
@@ -0,0 +1,102 @@
+- [Orders - impossible to extend order grid query from module](https://github.com/PrestaShop/PrestaShop/issues/24911)
+
+### Display regressions
+
+- [Buttons are not well displayed](https://github.com/PrestaShop/PrestaShop/issues/26040)
+- [Buttons are not well displayed](https://github.com/PrestaShop/PrestaShop/issues/25740)
+- [Buttons are not well displayed](https://github.com/PrestaShop/PrestaShop/issues/25980)
+- [Floating button over some elements on migrated pages and prevent to save form](https://github.com/PrestaShop/PrestaShop/issues/25423)
+
+
+## Front office regressions
+
+- [Problem when adding images with different ratio](https://github.com/PrestaShop/PrestaShop/issues/25853)
+- [After upgrade, theme must be updated to be able to pay into the final step](https://github.com/PrestaShop/PrestaShop/issues/26274)
+- [Simple anchors does not work on CMS page](https://github.com/PrestaShop/PrestaShop/issues/26326)
+- [The add to cart button is not working when clicking on the icon cart](https://github.com/PrestaShop/PrestaShop/issues/26433)
+- [Product disabled - an exception is displayed when changing combination or increasing / decreasing a quantity](https://github.com/PrestaShop/PrestaShop/issues/25891)
+- [Product comment structured data AggregateRating isn't working](https://github.com/PrestaShop/PrestaShop/issues/25372)
+- [Shopping cart page - An exception is displayed when a product added to the cart is transformed from simple product to a product with combinations](https://github.com/PrestaShop/PrestaShop/issues/25942)
+- [Product structured data json-ld problem when price is not displayed](https://github.com/PrestaShop/PrestaShop/issues/25371)
+- [Search with semicolon gives no result. Search differs from earlier versions of find()](https://github.com/PrestaShop/PrestaShop/issues/25394)
+- [An exception is displayed when a product added to the cart is transformed from simple product to a product with combinations](https://github.com/PrestaShop/PrestaShop/issues/25942)
+
+### Display regressions
+
+- [The product-flags are displayed in the same row with no space](https://github.com/PrestaShop/PrestaShop/issues/25272)
+- [There is an horizontal scrollbar on the homepage under 991px and the carousel overflows the content](https://github.com/PrestaShop/PrestaShop/issues/25459)
+- [The search button in the header is not responsive on mobile](https://github.com/PrestaShop/PrestaShop/issues/25133)
+- [Featured product not well displayed with responsive](https://github.com/PrestaShop/PrestaShop/issues/25283)
+
+
+## Module regressions
+- [Bad display in autoupgrade module](https://github.com/PrestaShop/PrestaShop/issues/25989)
+- [Ps_banner module fixed width and height value](https://github.com/PrestaShop/PrestaShop/issues/25852)
+
+
+## Acknowledgements
+
+Contributors to this patch version, from both the Core team and the community at large:
+
```

Comment:
This part is missing?

Reply:
Yes, I am waiting the list from @atomiix